collections/_posts/2022-11-23-FedAMP_Personalization_in_cross_silo_Federated_learning.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: review
+title: "FedAMP: Personalized Cross-Silo Federated Learning on Non-IID Data"
+tags:  federated learning, personalization, cross-silo
+author: "Manthe Matthis"
+cite:
+    authors: "Yutao Huang, Lingyang Chu, Zirui Zhou, Lanjun Wang,
+Jiangchuan Liu, Jian Pei, Yong Zhang"
+    title:   "Personalized Cross-Silo Federated Learning on Non-IID Data"
+    venue:   "AAAI 2021"
+pdf: "https://arxiv.org/pdf/2007.03797.pdf"
+---
+
+# Highlights
+
+* The authors propose a new federated personalization method,
+* The goal is to facilitate pairwise collaborations between institutions with similar data,
+* The method is based on an alternation between optimization of local losses and of a message passing regularizer,
+* The method can be applied to any supervised learning task (classification, segmentation, ...).
+
+# Method
+
+## Problem definition
+Consider a federated learning problem with $$M$$ institutions, each with $$n_m$$ samples.
+* Let's note the overall personalized objective function, with $$W:= [w_i]_{i<M}$$ the matrix of personalized model's parameters $$w_i\in\mathbb{R}^d$$
+
+$$ \mathcal{F}(W) := \sum_{i=1}^MF_i(w_i)$$
+
+* The message passing regularizer 
+
+$$ \mathcal{A}(W):= \sum_{i<j}^MA(||w_i-w_j||^2)$$
```

Comment:
I find the bullet list a bit confusing here. I would suggest to simply make the text flow around the equations. Something like:
```suggestion

Given $$W:= [w_i]_{i<M}$$ the matrix of personalized model's parameters $$w_i\in\mathbb{R}^d$$, let's define the overall personalized objective function:

$$ \mathcal{F}(W) := \sum_{i=1}^MF_i(w_i)$$

and the message passing regularizer:

$$ \mathcal{A}(W):= \sum_{i<j}^MA(||w_i-w_j||^2)$$
```